test/fetch-test.js

```diff
@@ -0,0 +1,66 @@
+const chai = require('chai');
+const spies = require('chai-spies');
+
+chai.use(spies);
+
+const expect = chai.expect;
+const APIRequests = require('../src/Fetch');
+
+describe('APIRequest', function() {
+
+  before(function() {
+    global.APIRequests = {};
+    chai.spy.on(APIRequests, ['fetchData'], () => {});
+  });
+  
```

Comment:
May want an afterEach

Reply:
I am not sure what an afterEach does. I will have to look it up. Thanks for the tip!